src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -2041,6 +2041,7 @@ class AlterSequenceOptionsSegment(BaseSegment):
             Sequence("MAXVALUE", Ref("NumericLiteralSegment")),
             Sequence("NO", "MAXVALUE"),
         ),
+        Sequence("SEQUENCE", "NAME", Ref("SequenceReferenceSegment")),
```

Comment:
You mention in the issue that this isn't in the docs - for future readers, can you add an inline comment above this mentioning that this isn't documented?

Reply:
@WittierDinosaur on it now :)

Reply:
good shout as it will stop future PRs from thinking it's a mistake and removing. I've added that comment in now.